image/scanner/rhel/konflux.Dockerfile

```diff
@@ -10,6 +10,9 @@ ARG SCANNER_TAG
 RUN if [[ "$SCANNER_TAG" == "" ]]; then >&2 echo "error: required SCANNER_TAG arg is unset"; exit 6; fi
 ENV RELEASE_TAG="${SCANNER_TAG}"
 
+# TODO(ROX-27054): Remove the redundant strictfipsruntime option if one is found to be so
```

Comment:
I don't _love_ the way this comment reads, but it's the best I could come up with.